AutoML/Cloud-Client

Found in #2937
https://source.cloud.google.com/results/invocations/f7079bef-8021-4a51-b88a-24740f041202/targets/cloud-devrel%2Fjava%2Fjava-docs-samples%2Fjava11%2Fpresubmit/log

# Java 11
```
------------------------------------------------------------
- testing automl/cloud-client
------------------------------------------------------------
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 302.896 s <<< FAILURE! - in com.example.automl.ImportDatasetTest
[ERROR] com.example.automl.ImportDatasetTest.testImportDataset  Time elapsed: 302.894 s  <<< ERROR!
java.util.concurrent.CancellationException: Task was cancelled.

[ERROR] com.example.automl.ImportDatasetTest.testImportDataset  Time elapsed: 302.895 s  <<< ERROR!
java.util.concurrent.ExecutionException:
com.google.api.gax.rpc.FailedPreconditionException: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: No other operations should be working on projects/233822521623/locations/us-central1/datasets/TEN1435998469759172608.
Currently running operations on the dataset:
	projects/233822521623/locations/us-central1/operations/TEN3671401816295211008
Call ListOperations API to learn more details.
	at com.google.common.util.concurrent.AbstractFuture.getDoneValue(AbstractFuture.java:564)
	at com.google.common.util.concurrent.AbstractFuture.get(AbstractFuture.java:545)
	at com.google.common.util.concurrent.FluentFuture$TrustedFuture.get(FluentFuture.java:86)
	at com.google.common.util.concurrent.ForwardingFuture.get(ForwardingFuture.java:62)
	at com.google.api.gax.longrunning.OperationFutureImpl.get(OperationFutureImpl.java:127)
	at com.example.automl.DeleteDataset.deleteDataset(DeleteDataset.java:44)
	at com.example.automl.ImportDatasetTest.tearDown(ImportDatasetTest.java:82)

Caused by: com.google.api.gax.rpc.FailedPreconditionException: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: No other operations should be working on projects/233822521623/locations/us-central1/datasets/TEN1435998469759172608.
Currently running operations on the dataset:
	projects/233822521623/locations/us-central1/operations/TEN3671401816295211008
Call ListOperations API to learn more details.

Caused by: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: No other operations should be working on projects/233822521623/locations/us-central1/datasets/TEN1435998469759172608.
Currently running operations on the dataset:
	projects/233822521623/locations/us-central1/operations/TEN3671401816295211008
Call ListOperations API to learn more details.
	at io.grpc.Status.asRuntimeException(Status.java:533)
	... 15 more

[ERROR] Errors:
[ERROR] com.example.automl.ImportDatasetTest.testImportDataset
[ERROR]   Run 1: ImportDatasetTest.testImportDataset:89 » Cancellation Task was cancelled.
[ERROR]   Run 2: ImportDatasetTest.tearDown:82 » Execution com.google.api.gax.rpc.FailedPrecond...
[ERROR] Tests run: 35, Failures: 0, Errors: 1, Skipped: 0
```

Looks like bad test design.  Can't run two of them at the same time. :(
@stephaniewang526

--- a/automl/cloud-client/pom.xml
+++ b/automl/cloud-client/pom.xml
@@ -40,7 +40,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>libraries-bom</artifactId>
-        <version>5.2.0</version>
+        <version>5.5.0</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>

